### ISSUE TITLE
Definitions 

### ISSUE THREAD

**Issue by fameri** [HediKarray/COMAIN_ROMAIN#2]
work management processes don't have definitions.

**Comment by fameri** [comment]
I agree with you Hedi. A Maintenance Action is a compose of one or more elementary Maintenance Tasks.

**Comment by fameri** [comment]
also we need to make sure to use 'definition' instead of 'isDefinedB' annotation property for all definitions.

**Comment by HediKarray** [comment]
i added an annotation property "definition" 
we need to update it for all classes. 
Also, add definition of bfo classes

**Comment by mhodki** [comment]
When I go + on annotations I do not see a definition option only an isDefinedBy option

**Comment by HediKarray** [comment]
My students are until now autonomous, they are programming without asking lot of questions. :)

**Comment by mhodki** [comment]
What goes in the Reasoning section 6.1 and 6.3 Possible and impossible extensions

**Comment by mhodki** [comment]
I agree. Can you write something?

Sent from my IPhone

On 16 May 2018, at 17:22, tlouge <notifications@github.com<mailto:notifications@github.com>> wrote:


Do we need to explain (a little, main concepts/relations) in the reasoning section how do we use the ontology to answer the competency questions? I think yes, but just to be sure...
